File: app/src/main/java/com/example/contactapp/iContact.java
package com.example.contactapp;

import java.util.ArrayList;

public interface iContact {
    ArrayList<Contact> Contactos = new ArrayList<Contact>();
    void add (Contact contacto);
    String phone (int numero);
    String[] mail (String destinatario);
    ArrayList<String> getlistas ();



}

File: app/src/main/java/com/example/contactapp/iContact.kt
package com.example.contactapp

import java.util.ArrayList

interface iContact {
    fun add(contacto: Contact)
    fun phone(numero: Int): String
    fun mail(destinatario: String): Array<String>
    fun getlistas(): ArrayList<String>

    companion object {
        val Contactos = ArrayList<Contact>()
    }


}
